falcon/request.py

```diff
@@ -1160,6 +1162,8 @@ class RequestOptions(object):
             request's content type is
             *application/x-www-form-urlencoded* (default ``False``). In
             this case, the request's content stream will be left at EOF.
+        auto_parse_query_string_lists: Set to ``False`` in order to disable
```

Comment:
How about: `This might be helpful if comma encoding lists is not easy in the language/framework talking to the api.`?